src/Stratis.Bitcoin.Features.Miner/PowMining.cs

```diff
@@ -27,6 +28,20 @@ public ReserveScript(Script reserveFullNodeScript)
 
     public class PowMining : IPowMining
     {
+        /// <summary>The async loop we need to wait upon before we can shut down this feature.</summary>
+        private IAsyncLoop asyncLoop;
```

Comment:
maybe name it smth like `miningLoop` for the consistency? (in PoSMinting it's called stakingLoop so it makes sense to follow the same pattern here)

Reply:
Renamed (was gonna do variable renaming in next PR but we can do this one here.) 👍